.github/workflows/docker-image-package-tester.yml

```diff
@@ -0,0 +1,107 @@
+name: Docker Linux Packaging Tester CI
```

Comment:
Maybe more simply `Linux Package Tests`, or be more explicit that it's leveraging .NET `.NET Linux Package Tests`?

Reply:
I can actually move this into `package-linux.yml`?

Reply:
I'm fine with that.